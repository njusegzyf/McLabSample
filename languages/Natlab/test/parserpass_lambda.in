a = @() 1
b = @(x) x
c = @(x, y) y - x
d = @(x, y) (y - x)
e = @(x, y) (@(x) x)
f = @(x)@(y) x + y