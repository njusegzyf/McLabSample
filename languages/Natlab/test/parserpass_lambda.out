1 1
6 18
a = (@() 1)
b = (@(x) x)
c = (@(x, y) (y - x))
d = (@(x, y) (y - x))
e = (@(x, y) (@(x) x))
f = (@(x) (@(y) (x + y)))
